### PR TITLE
Fixed the token cannot be refreshed

### PR DESCRIPTION
**Pull request by WillardHu**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Fixed the token cannot be refreshed. 
This PR needs to be rebased to release-0.18 and release-0.19

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

File: cloud/pkg/cloudhub/servers/httpserver/pre_server.go [hunk 1]
```diff
@@ -159,32 +159,38 @@ func createCertsToSecret(ctx context.Context) error {
...
+					return
 				}
+				klog.Info("token refreshed successfully")
 			case <-ctx.Done():
 				break
 			}
 		}
 	}()
-	klog.Info("Succeed to creating token")
+	klog.Info("token created successfully")
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/pre_server.go:180]: Do we need to add log level here and L174?

Comment (WillardHu) [reply cloud/pkg/cloudhub/servers/httpserver/pre_server.go:180]: When the token is not refreshed, we can search the history log by information. The frequency of printing here is very low.

Comment (fisherxu) [reply cloud/pkg/cloudhub/servers/httpserver/pre_server.go:180]: Sounds good.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**